src/constants/index.js

```diff
@@ -25,6 +25,7 @@ export { default as usageConsolidation } from './usageConsolidationReduxStateSha
 export { default as ucCredentialsReduxStateShape } from './ucCredentialsReduxStateShape';
 export { default as sortOrders } from './sortOrders';
 export { default as COVERAGE_STATEMENT_VALUE_MAX_LENGTH } from './COVERAGE_STATEMENT_VALUE_MAX_LENGTH';
+export { default as CONTRIBUTOR_FIELD_VALUE_MAX_LENGTH } from './CONTRIBUTOR_FIELD_VALUE_MAX_LENGTH';
```

Comment:
Please, rename file as `./contributorFieldValueMaxLength` and please, fix it for `coverageStatementValueMaxLength`, I didn't catch it before. It will be not an error for sonar, because it's not case sensitive.